datasets/thelook_ecommerce/pipelines/_images/run_thelook_kub/fake.py

```diff
@@ -723,7 +730,7 @@ def __post_init__(self, order_item=None):
         if order_item.is_sold is False:
             self.created_at = created_at(datetime.datetime(2020, 1, 1))
             self.sold_at = None
-        self.cost = PRODUCT_BY_ID_DICT[self.product_id]["cost"]
+        self.cost = (float(PRODUCT_BY_ID_DICT[self.product_id]["cost"]) *  (random.randrange(25, 90)/100))
```

Comment:
Not sure if we should use cost to calculate a new value of cost? Because you meant to make the cost lower than the price.

Move the calculation to where you're generating the other attributes, so you can simply do
```
self.cost = PRODUCT_BY_ID_DICT[self.product_id]["cost"]
```

Reply:
We can also extract this part `random.randrange(25, 90)/100)` into a more descriptive function.